content/help.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Help
+---
+
+There are several avenues to get help with cloud.gov:
+
+## I am interested in using cloud.gov.
+
+Fill out our [contact form](https://cloud.gov/#contact).
+
+## I have a cloud.gov account and need help.
+
+Email [cloud-gov-support@gsa.gov](mailto:cloud-gov-support@gsa.gov).
+
+## I am 18F staff and need help.
+
+Talk to us in [#cloud-gov-support](https://18f.slack.com/messages/cloud-gov-support).
+
+## I found a problem with the documentation.
+
+[Open an issue on GitHub](https://github.com/18F/cg-docs/issues/new).
+
+## I'm interested in working on cloud.gov.
+
+Check out [18F's jobs site](https://pages.18f.gov/joining-18f/), and come talk to us in [our #devops-public Slack channel](https://chat.18f.gov/).
```

Comment:
@mogul Does all of the information here seem accurate, for the current state of things? Any sorts of requests you can think of that I didn't mention? Is there too much?